resources/puppetlabs/lein-ezbake/template/global/ext/cli/stop.erb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set +e
+
+pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
```

Comment:
I did a survey across a few OSes - CentOS 6/7, Ubuntu 12.04/14.04/16.04, SLES 11/12, Debian 8 - and for each, I saw that `procps` was providing pgrep.  It appears we have already been using `pgrep` for RedHat derivatives in shipping init scripts but nothing for Debian, AFAICT.

I agree with the suggestion - just want to make sure I understand what you're asking me to do here.  Would I add `procps` as a `Requires` in the "ezbake.spec.erb" files for rpm and as a `Depends` in the control.erb for deb - for both foss and pe?  Anywhere else?